src/ExceptionHandler.php

```diff
@@ -31,7 +34,8 @@ public static function renderAsApiResponse(Throwable $e)
 
             return ApiResponse::error($traces ? ['_traces' => $traces] : [])
                 ->message($self->getMessage($e))
-                ->statusCode($self->getStatusCode($e));
+                ->statusCode($self->getStatusCode($e))
+                ->toResponse($request);
```

Comment:
i am curious can we do something like this:

```php
$apiResponse = ApiResponse::error(); //...

return response($apiResponse);
```

nevertheles, this one is also oke. thank you mas!